cmd/release-controller/sync_release.go

```diff
@@ -41,7 +41,7 @@ func (c *Controller) ensureReleaseJob(release *Release, name string, mirror *ima
 		job.Annotations[releaseAnnotationGeneration] = strconv.FormatInt(release.Target.Generation, 10)
 		job.Annotations[releaseAnnotationReleaseTag] = mirror.Annotations[releaseAnnotationReleaseTag]
 
-		klog.V(2).Infof("Running release creation job %s/%s for %s", job.Namespace, job.Name, name)
+		klog.V(2).Infof("Running release creation job %s/%s for %s", c.jobNamespace, job.Name, name)
```

Comment:
Why the change?  I prefer to output the content from the actual thing created

Reply:
This was really just an ancillary fix to address an odd message that I had been seeing all day:
```
I0710 03:19:50.855823       1 sync_release.go:44] Running release creation job /4.9.0-0.ci-2021-07-10-031232 for 4.9.0-0.ci-2021-07-10-031232
```
I couldnt find any indication that the namespace was being set anywhere and these particular jobs are always run in `c.jobNamespace`.

Reply:
heh.  That seems.... concerning.  How about fixing it in the other direction by assigning the job.Namespace correctly?

Reply:
We agreed on a followup